src/main/resources/META-INF/rewrite/spring-boot-30.yml

```diff
@@ -200,3 +201,28 @@ recipeList:
       methodPattern: org.springdoc.core.models.GroupedOpenApi.Builder addOpenApiCustomiser(..)
       newMethodName: addOpenApiCustomizer
       matchOverrides: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.boot3.MigrateDropWizardDependencies
+displayName: Migrate dropWizard dependencies to Spring Boot 3.x
+description: Migrate dropWizard dependencies to the new artifactId, since these are changed with Spring Boot 3.
+tags:
+  - spring
+  - boot
+  - dropwizard
+recipeList:
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: com.codahale.metrics.servlet
+      newPackageName: com.codahale.metrics.servlets
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: io.dropwizard.metrics.servlet
+      newPackageName: io.dropwizard.metrics.servlets
```

Comment:
From [this comment](https://github.com/openrewrite/rewrite-spring/issues/506#issuecomment-2008914623) we should change how packages are renamed. Other than that we're close. Thanks for getting this started!